CHANGELOG.md

```diff
@@ -37,10 +37,10 @@
   - Component `spiral/annotations` is removed. Use `spiral/attributes` instead.
   - Added return type `void` to a methods `publish`, `publishDirectory`, `ensureDirectory` in `Spiral\Module\PublisherInterface` interface.
   - Removed `Spiral\Http\SapiDispatcher` and `Spiral\Http\Emitter\SapiEmitter`. Please, use package `spiral/sapi-bridge` instead.
-  - Bootloader `Spiral\Bootloader\Http\DiactorosBootloader` moved to the `Spiral\Http\Bootloader\DiactorosBootloader`.
+  - Bootloader `Spiral\Bootloader\Http\DiactorosBootloader` is removed. Please, use package `spiral/nyholm-bridge` instead.
```

Comment:
Could you add more info about NyholmBootloader here?

Reply:
Added